app/assets/stylesheets/application.scss

```diff
@@ -1 +1,111 @@
-@import "bootstrap"
+@import "bootstrap";
+
+// --- BODY --- //
+a {
+    color: #fff;
+    margin: 25px;
+}
+
+a:hover {
+    text-decoration: none;
+    color: #000;
+}
+
+h3 {
+    text-align: center;
+}
+.nav {
+    padding: 15px 0px;
+}
+
+#home-img {
+    width: 100%;
+    height: auto;
+}
+
+// --- PROFILE VIEW --- //
+#profile-contents {
+    display: flex;
+    flex-direction: wrap;
+}
+#user-profile-body {
+    text-align: center;
+    border: 1px solid #478f92;
+    margin: 15px 30px;
+    width: 500px;
+    padding: 15px 10px;
+}
+
+.avatar {
+    width: 200px;
+    height: auto;
+}
+
+// --- PROFILE EDIT --- //
+.submit-btn {
+    background-color: #d0bb65;
+    border: none;
+    border-radius: 20px;
+}
+
+
+// --- EMERGENCY CONTACT --- //
+#index-body {
+    width: 300px;
+    border: 2px solid #92477d;
+    margin: 15px 30px;
+}
+
+.e-contact {
+    border: 2px solid #478f92;
+    padding: 15px;
+    width: 296px;
+}
+
+.edit-btn {
+    background-color: #d065b3;
+    border: none;
+    border-radius: 20px;
+}
+
+.delete-btn {
+    background-color: #d0bb65;
+    border: none;
+    border-radius: 20px;
+}
+
+.add-btn {
+    background-color: #92477d;
+    border: none;
+    border-radius: 20px;
+}
```

Comment:
FYI, since you've already got `bootstrap`, you can actually use their different [Bootstrap Button Styles](https://www.w3schools.com/bootstrap4/bootstrap_buttons.asp), like `btn-primary`, `btn-secondary`, and `btn-default` -- as well as `btn-lg` or `btn-sm`

Reply:
ew bootstrap colors